CRM/Utils/Check/Component.php

```diff
@@ -62,11 +62,15 @@ public function checkAll() {
    * Check if file exists on given URL.
    *
    * @param string $url
-   * @param float $timeout
+   * @param float|bool $timeout
    *
    * @return bool
    */
-  public function fileExists($url, $timeout = 0.50) {
+  public function fileExists($url, $timeout = FALSE) {
```

Comment:
should we rename it to timeoutOverride?

Reply:
@seamuslee001 see above - would that be clearer / more accurate?

Reply:
@eileenmcnaughton i don't think so i think its fine just as is